src/BookingForm.js

```diff
@@ -0,0 +1,195 @@
+import React, { useState } from "react";
+import SearchResults from "./SearchResults.js";
+import TableRow from "./TableRow.js";
+function BookingForm({ handleNewBooking }) {
+  const [isSubmitClicked, setSubmitClicked] = useState(false);
+  const [newBookingInfo, setNewbookingInfo] = useState({
+    id: "0",
+    title: "",
+    firstName: "",
+    surname: "",
+    email: "",
+    roomId: "",
+    checkInDate: "",
+    checkOutDate: "",
+    items: []
+  });
+
+  function handleSelect(event) {
+    let newId = 5;
+    newId++;
+    const updatedCustomerInfo = {
+      id: newId,
+      title: event.target.value
+    };
+    setNewbookingInfo(updatedCustomerInfo);
+  }
+
+  function handleChange(event) {
+    const updatedCustomerInfo = {
```

Comment:
This looks good - updating the state of the controlled component only -namely, the fields of the form